service/gcs/prot/protocol.go

```diff
@@ -646,6 +646,14 @@ type MessageResponseBase struct {
 	ErrorRecords []ErrorRecord `json:",omitempty"`
 }
 
+// GetBase assigns `mrp` if `nil` and returns `mrp`.
+func (mrp *MessageResponseBase) GetBase() *MessageResponseBase {
+	if mrp == nil {
+		mrp = &MessageResponseBase{}
```

Comment:
This gets thrown away--why do it?

Reply:
Just switched from pointers as the base message to avoid this.